gsa_admin.py

```diff
@@ -549,7 +576,7 @@ def exportAllUrls(self, out):
       request = urllib2.Request(self.baseURL, param)
       result = self._openurl(request)
       if self.is72:
-        generating_msg = '<input type="submit" name="generate" id="generate" disabled class="hb-r-N nd-Ld-re" value="Generating...">'
+        generating_msg = '<input type="submit" name="generate" id="generate" disabled class="hb-r-N od-Md-se" value="Generating...">'
```

Comment:
This is related a change in 7.4 when new feature to export more data was added. This likely changed class name. When you open Index > Diagnostics > Export URLs and inspect "Generated the gzip file" control, you will see the class name is different. It may require condition rather than changing the class.